baseplate/sidecars/event_publisher.py

```diff
@@ -130,16 +151,18 @@ def publish(self, payload: SerializedBatch) -> None:
         for _ in RetryPolicy.new(budget=MAX_RETRY_TIME, backoff=RETRY_BACKOFF):
             try:
                 with self.metrics.timer("post"):
```

Comment:
Since we're no longer publishing statsd metrics, this can probably be removed along with other references to self.metrics.